templates/ingress.yaml

```diff
@@ -193,7 +193,7 @@ spec:
           {{- if .Values.authSidecar.enabled  }}
           - path: /{{ .Release.Name }}/dags
           {{- else }}
-          - path: /{{ .Release.Name }}/dags/(upload|download)(/.*)?
+          - path: /{{ .Release.Name }}/dags/(upload|downloads|healthz)(/.*)?
```

Comment:
Is there any good reason to make downloads and healthz public?

Reply:
@pgvishnuram It's easy for us to use this endpoint to check the dag-server validations in our automation